app/src/components/market/sections/market_buy/market_buy.tsx

```diff
@@ -227,15 +246,22 @@ const MarketBuyWrapper: React.FC<Props> = (props: Props) => {
       />
       <GridTransactionDetails>
         <div>
-          <WalletBalance
-            onClick={() => {
-              setAmount(collateralBalance)
-              setAmountToDisplay(formatNumber(formatBigNumber(collateralBalance, collateral.decimals), 5))
-            }}
-            symbol={collateral.symbol}
-            value={formatNumber(formatBigNumber(collateralBalance, collateral.decimals), 5)}
-          />
-          <ReactTooltip id="walletBalanceTooltip" />
```

Comment:
Did you mean to delete the tooltip here? Because we still have the icon for it, just not the tip itself

Reply:
yes, remove tooltip